src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -5116,7 +5115,7 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
         }
       }
 
-      def typedTypeSelectionQualifier(tree: Tree, pt: Type = AnyRefTpe) =
+      def typedTypeSelectionQualifier(tree: Tree, pt: Type /*= AnyRefTpe*/) =
```

Comment:
documentary default, removing unused default.

Reply:
Happy to have the default removed

Reply:
Made non-documentary.